components/OrganizationEdit/OrganizationEdit.js

```diff
@@ -240,17 +240,21 @@ class OrganizationEdit extends Component {
     })
 
     updateOrganization(organization).then(response => {
+      let success = !response.hasOwnProperty('error')
       this.setState({
         hasSubmit: true,
-        submitResult: !response.hasOwnProperty('error')
+        submitResult: success,
```

Comment:
rename variable above to `submitResult` to take advantage of ES6 object literal shorthand?
```js
this.setState({
  hasSubmit: true
  submitResult,
  changesExist: !submitResult
})
```